src/electron/lib/Blix.ts

```diff
@@ -60,14 +60,8 @@ export class Blix {
     // this._aiManager = new AiManager(mainWindow);
     this._projectManager = new ProjectManager(mainWindow);
 
-    // Add subscribers
-    const graphSubscriber = new IPCGraphSubscriber();
-
-    graphSubscriber.listen = (graphId: UUID, newGraph: UIGraph) => {
-      mainWindow.apis.graphClientApi.graphChanged(graphId, newGraph);
-    };
-
-    this._graphManager.addAllSubscriber(graphSubscriber);
+    this.initSubscribers();
```

Comment:
The graph IPC subscriber is kind of an special case subscriber. Originally the graph subscriber was created in GraphApi.ts. I don't think it belongs here though because the MainApis is only there to facilitate communication invoked from the frontend whereas the GraphSubscriber sends the new state to the frontend through the Window/Client API channels. So, for now I think when Blix instantiates it is best to also instantiate all the special subscribers then and there.

Reply:
Yea I guess this is just a matter of taste; personally I prefer that each module is responsible for its own subscriber. If we move the 'special' subscribers to Blix.ts then it seems like things might feel a bit disorganized. As an example there might be cases where a module might want to swap out the handler `listen` function internally, which would feel a bit out of place if the subscriber initialization is done first in Blix.
But yea both work and its easy enough to change after the fact.